### ISSUE TITLE
Add ability to see vertex document when selected in graph viewer

### ISSUE THREAD

**Issue by robross0606** [arangodb/arangodb#11239]
The graph viewer from the query window is almost useless as it currently stands.  All it shows you is the `_key `of each vertex, with zero other information available or accessible.  Even if it showed the `_id `this would be more useful for tracking documents across collections.  What would make it _actually_ usable would be an optional window or popup where it showed the JSON of the currently selected vertex.  Another useful feature would be to offer an option of showing a different field label on vertices.  But at least give us the full `_id` and not just the `_key`.

**Comment by maxkernbach** [comment]
Hi @robross0606,

The JSON of the selected vertex in the graph viewer can be accessed by clicking the "pencil" icon.

![image](https://user-images.githubusercontent.com/9152640/76407191-3e986480-638b-11ea-8839-212ce7f4fce7.png)

#5833 is open feature request which already discusses this topic. Could you please use the referenced issue to share your ideas?

**Comment by robross0606** [comment]
I said "the graph viewer FROM THE QUERY WINDOW". Not the "Graphs" viewer.  The ticket you referenced is for a completely different interface with a totally different feature set.

**Comment by robross0606** [comment]
This is NOT a duplicate of #5833.

**Comment by maxkernbach** [comment]
Sorry, I missed this point.
Reopening as a new feature request.

**Comment by maxkernbach** [comment]
However, there is the following workaround:

The graph view on from the query window offers the option to open the resulting Graph in the **Graph Viewer** as you can see in the bottom right corner of the attached screenshot:

 ![Screenshot from 2020-03-11 13-23-22](https://user-images.githubusercontent.com/9152640/76416645-bff7f300-639b-11ea-9972-b14f74b0e844.png)

Then, in the **Graph Viewer** you can use the pencil icon as shown in my first comment.

Is this helpful to you?

**Comment by robross0606** [comment]
No one running here is seeing a pencil icon on vertices when clicked. Is there a setting somewhere or a trick to this?

**Comment by maxkernbach** [comment]
There is **no** pencil icon in the **Query editor result Graph view**, but there is a pencil icon in the **Graph Viewer** (which can be opened by clicking on the **Graph Viewer** button in the previous screenshot).

Once the **Graph Viewer** is open, you can leftclick a vertex and a pencil icon is shown. There is no setting to enable this.

![image](https://user-images.githubusercontent.com/9152640/76417614-7ad4c080-639d-11ea-8105-09c6c93c3f04.png)

This screenshot is taken using the latest release 3.6.1 on Ubuntu. In case this does not work for you, could you please share information about your environment (see Issue template)?

**Comment by robross0606** [comment]
I understand that.  I'm saying we don't see it on the Graph Viewer either.  We are on 3.5.4 and haven't switched to 3.6.x line because a large percentage of our queries broke when we upgraded.  In one instance, our actual database was corrupted by the upgrade process.  Is this only available on 3.6.x and above?

**Comment by maxkernbach** [comment]
No, it is not only available in 3.6.x. I just double-checked 3.5.4 and it worked as well.

Maybe this is related to some Browser/Caching issue?
I am using Chrome on Ubuntu 18.04.3 LTS, which one are you using? Could you test it with another browser?

**Comment by maxkernbach** [comment]
In addition version 3.6.0 had a bug affecting `splice-subqueries` optimization rule, which caused a lot of queries to fail. This is fixed in 3.6.1.

Did you previously upgrade to 3.6.0 or also have issues when upgrading to 3.6.1?

**Comment by robross0606** [comment]
We had previously upgraded to 3.6.0. It caused a LOT of chaos so we're going to be far more cautious about future upgrades.  We haven't tried 3.6.1 yet.

**Comment by robross0606** [comment]
Okay, I was finally able to see what you're talking about with the pencil icon.  It's something but not really quite what I was talking about.  In order to see information, you'd have to take three clicks and two windows away from your original query.  It's better than nothing but really cumbersome when trying to debug a query and/or data.

**Comment by maxkernbach** [comment]
Thanks for your feedback. I agree that the usability could definitely be improved here.
